drivers/telescope/lx200_10micron.h

```diff
@@ -138,9 +138,7 @@ class LX200_10MICRON : public LX200Generic
     bool SetTLEtoFollow(const char *tle);
     bool SetTLEfromDatabase(int tleN);
     bool TrackSat();
-    bool CalculateTrajectory(int year, int month, int day, int hour, int minute, int nextpass, ln_date date_pass);
-    struct ln_date today;
-    struct ln_date date_pass;
+    bool CalculateSatTrajectory(char *start_pass, char *end_pass);
```

Comment:
let's use the same names as in the implementation

Reply:
Done